src/server/utils.js

```diff
@@ -1,5 +1,10 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+
 import { getEnv } from 'src/utils/env';
 
+import withSSRData from '../shared-components/withSSRData';
```

Comment:
So `withSSRData` can be move from `shared-components` to `src/services/initial-ssr-data/`